tests/test_config.py

```
@@ -0,0 +1,137 @@
+        ),
+        "I'm dealing with a very challenging customer situation": (
+            "Vague query requiring context expansion"
+        ),
+    }
+
+    # Combined dictionary built once at class definition time (immutable)
+    ALL_QUERIES: Dict[QueryText, Tuple[QueryCategory, QueryDescription]] = {
+        **{query: (CATEGORY_BASIC, description) for query, description in BASIC_QUERIES.items()},
+        **{query: (CATEGORY_EDGE_CASE, description) for query, description in EDGE_CASE_QUERIES.items()},
+        **{query: (CATEGORY_STRESS_TEST, description) for query, description in STRESS_TEST_QUERIES.items()}
+    }
```

Ruff (0.13.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)